content/learn/migration-guides/0.10-0.11/_index.md

```diff
@@ -1319,6 +1319,15 @@ Flip `EnvironmentMapLight` maps if needed to match how they previously rendered
 
 When sampling from the point light shadow cubemap, use the (expected) light to fragment direction vector but negate the z coordinate. Previously, you would have used the fragment to light direction vector.
 
+### [Add `Aabb` calculation for `Sprite`, `TextureAtlasSprite` and `Mesh2d`](https://github.com/bevyengine/bevy/pull/7885)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">Rendering</div>
+</div>
+
+- 2D entities with negative z values are now culled by frustum culling.
+- This will get fixed by the next patch release.

```

Comment:
```suggestion
- 2D entities are now subject to frustum culling, check your 2D camera's z coordinate and projection `far` if some of them are not rendered anymore
- In particular, 2D entities with negative z values are now culled by frustum culling with the `default` `Camera2dBundle`.
- That last point will get fixed in version `0.11.1`.
```

1. Mention frustum culling in general and what to look at to fix the apparent issue
2. Mention that the negative z-value thing is only with the default camera
3. Mention the "next patch version" explicitely

Reply:
Sorry about that, pretty new to writing docs.

Thanks for your suggestions!

Reply:
No problem, thank you for doing this PR :purple_heart: